package.json

```diff
@@ -105,7 +104,9 @@
     "terser": "^5.14.2",
     "jpeg-js": "^0.4.4",
     "got": "^11.8.5",
-    "loader-utils": "^2.0.4"
+    "loader-utils": "^2.0.4",
+    "**/@storybook/**/minimatch": "^3.0.5",
```

Comment:
I'm thinking we should start pinning resolutions that are for security concerns in this way rather than a global pin which makes it hard to resolve dependencies down the line and makes it unclear what were the upstream packages that were using it and why it needed to be pinned.

This helps limit the scope of these global resolution overrides, though the syntax is a bit confusing. Their docmentation doesn't explain it very well but the [RFC](https://github.com/yarnpkg/rfcs/blob/f1d11a23cc87c5e89d909521197e6a5028e05e05/implemented/0000-selective-versions-resolutions.md#package-designation) gives more info, our normal resolutions are actually aliases for `**/package` but if you want to further scope the packages the alias doesn't seem to work.

Reply:
I LOVE IT!! defining resolutions this way is veeeeery descriptive of why we are doing it, which will facilitate A LOT when we have to audit or mantain them, awesome!